pom.xml

```diff
@@ -96,6 +96,12 @@
             <version>2.1.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
i'm not sure if it's a good idea to includes sling-mock as test dependency in sling-api, it' some what a cyclic dependency. not sure if we have done this on other sling modules on which sling-mock depend in the same way.
but that's not a blocker as it has not affect on the main code or on a release.

Reply:
mmm i see your point, though 1. it's a test dependency 2. i won't tell _you_ it's the best mock for `Resource` API i can get - and that i need - in an elegant way

Reply:
but, it's true that while i'm not annoyed by using it to mock Resource API for testing stuff around value map, this opens the door for weird testing on Resource API...

Reply:
will fix that